### PR TITLE
Package aws-s3.1.1.0

### PR DESCRIPTION
**Pull request by andersfugmann**
### `aws-s3.1.1.0`

Client Library for simple S3 access.

The library implements methods to cp (get and put), rm (single and
multi) and ls. Also included is IAM bindings to get machine role and
security tokens.

---
* Homepage: https://github.com/andersfugmann/aws-s3
* Source repo: https://github.com/andersfugmann/aws-s3
* Bug tracker: https://github.com/andersfugmann/aws-s3/issues

---

:camel: Pull-request generated by opam-publish v0.3.5

### CODE REVIEW

File: packages/aws-s3/aws-s3.1.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+homepage: "https://github.com/andersfugmann/aws-s3"
+dev-repo: "git+https://github.com/andersfugmann/aws-s3"
+bug-reports: "https://github.com/andersfugmann/aws-s3/issues"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" { build }
+  "core" { >= "0.9" }
+  "async"
+  "async_extended"
+  "ounit"
```
Comment (rgrinberg) [inline packages/aws-s3/aws-s3.1.1.0/opam]: Should this be a test dependency perhaps?

Comment (andersfugmann) [reply packages/aws-s3/aws-s3.1.1.0/opam]: Currently unit tests reside in the same file as the module implementation in order to have access to internal functions that may not be exposed by the module signature - so unfortunately for now it cannot be a test dependency only.

**Comment by andersfugmann** [comment]
Build failed on 4.02.3 and 4.06. The library depends on core that is only available for 4.03. Also core is lacking a constraint to `{ < 4.06 }` since it cannot build on 4.06.

**Comment by avsm** [comment]
Core constraint has been fixed, thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**